src/app/app.component.scss

```diff
@@ -0,0 +1,54 @@
+@import url('https://fonts.googleapis.com/css2?family=ABeeZee&family=Comfortaa:wght@300;400;500;600;700&display=swap');
+
+.navigation{
+    display: flex;
+    flex-direction: row;
```

Comment:
The default value of flex-direction is row, therefore not needed to state it explicitly